examples/aggregator/module/rust/Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = "2018"
 license = "Apache-2.0"
 
 [lib]
-crate-type = ["cdylib"]
+crate-type = ["cdylib", "rlib"]
```

Comment:
Is `rlib` added to make the modules available to the integration tests? How is `oak/server/rust/oak_runtime/tests/integration_test.rs` working without it? What is its magic? 

Not so important, but perhaps `oak/server/rust/oak_runtime/tests/integration_test.rs` could be renamed to `integration_tests.rs` as well, to be consistent :)

Reply:
Oh! Just saw your description!

Reply:
I think the magic is that `rlib` is the default (and so not mentioned in `oak_runtime/Cargo.toml`; however, the examples need to be built for the Wasm target and that (I think) needs `cdylib` crate type.

Also, it looks like `integration_test.rs` is more canonical so I'll move to that in the examples.